FirebaseMessaging/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 10.3.0
+- [changed] Allow notification support on iOS 16 Simulator on Xcode 14 (Reference: Xcode 14 Release Notes -> Simulator -> New Features: https://developer.apple.com/documentation/xcode-release-notes/xcode-14-release-notes)
```

Comment:
Add (#9968) to the line to indicate this fixes GitHub issue #9968.

Also add the string "Fix #9968" to the GitHub PR description and GitHub will automatically close the issue when this PR merges to master.